helm_deploy/prisoner-content-hub-frontend/templates/ingress-old.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "prisoner-content-hub-frontend.fullname" . -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "prisoner-content-hub-frontend.labels" . | nindent 4 }}
+  annotations:
+    {{- range $k, $v := .Values.ingress.annotations }}
+    {{ $k }}: {{ tpl $v (dict "qualifier" $.Values.ingress.qualifier "Template" $.Template) | quote }}
+    {{- end}}
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/whitelist-source-range: {{ include "app.joinListWithComma" .Values.ingress.allowed | quote }}
+    external-dns.alpha.kubernetes.io/aws-weight: "0"
+    {{ if eq .Values.environmentName "development" }}
+    external-dns.alpha.kubernetes.io/set-identifier: prisoner-content-hub-frontend-{{ .qualifier }}-prisoner-content-hub-development-green
+    {{ else }}
+    external-dns.alpha.kubernetes.io/set-identifier: prisoner-content-hub-frontend-prisoner-content-hub-{{ .Values.environmentName }}-green
+    {{ end }}
+spec:
+  tls:
+  - hosts:
+  {{- range .Values.ingress.hosts }}
+  {{- $certSecret := .cert_secret -}}
+  {{- $domainPattern := .pattern }}
+    {{- range $.Values.ingress.prefixes }}
+      - {{ tpl $domainPattern (dict "prison" . "qualifier" $.Values.ingress.qualifier "Template" $.Template) }}
+    {{- end }}
+  {{- end }}
+  {{- with .Values.ingress.host }}
+      - {{ tpl .pattern (dict "qualifier" $.Values.ingress.qualifier "Template" $.Template) }}
+    {{ if .cert_secret }}secretName: {{ .cert_secret }}{{ end }}
+  {{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+    {{- $domainPattern := .pattern -}}
+    {{- range $.Values.ingress.prefixes }}
+    - host: {{ tpl $domainPattern (dict "prison" . "qualifier" $.Values.ingress.qualifier "Template" $.Template) }}
+      http:
+        paths:
+          - path: "/"
+            pathType: Prefix
+            backend:
+              service:
+                name: {{ $fullName }}
+                port:
+                  number: 80
+    {{- end }}
+  {{- end }}
+  {{- with .Values.ingress.host }}
+    - host: {{ tpl .pattern (dict "qualifier" $.Values.ingress.qualifier "Template" $.Template) }}
+      http:
+        paths:
+          - path: "/"
+            pathType: Prefix
+            backend:
+              service:
+                name: {{ $fullName }}
+                port:
+                  number: 80
+  {{- end }}
+{{- end }}
```

Comment:
This is a copy of the old ingress.yml file.  We need two versions so that we can deploy with no downtime.